recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,290 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
+
+build:
+  number: 0
+  skip: True  # [not py35]
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  build:
+    - python >3.5.0,<3.5.9
```

Comment:
This is quite restrictive. Are you sure this package won't work with Python >=3.5.9?

Reply:
No, but I tried this with Python 3.6 a couple of weeks ago and got all kinds of dependency failures, so I thought it would be safer to restrict it to Python 3.5.

Reply:
> No, but I tried this with Python 3.6 a couple of weeks ago and got all kinds of dependency failures

Probably b/c we don't have versions of all those on Python 3.6. They are quite restrictive too. I'll try to find which ones and see if we can do something about it.